flybywiresim/a32nx
# feat(atsu): Coupled atsu and DATALINK STATUS page

Fixes #[issue_no]

## Summary of Changes

Coupled the DATALINK STATUS page onto the atsu, per request of @svengcz.
Added new functions to atsu to let the DATALINK STATUS page retrieve data directly from the atsu.

Functions added:
DatalinkStatus and DatalinkMode, for both VHF3, HF and SATCOM.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): space_viking#0186

## Testing instructions

There really isn't too much to test in this as there visually shouldn't be any difference. 2nd line of VHF should read ATC/AOC if hoppie is enabled and AOC ONLY if disabled. SATCOM should read "NOT INSTALLED" if satcom is disabled in the EFB and "DLK NOT AVAIL" if satcom is enabled.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artefact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
